File: src/test/java/destiny/data/gauquelin/GPersonAnglePowerTest.java
/**
 * @author smallufo 
 * Created on 2009/3/21 at 上午 1:04:35
 */ 
package destiny.data.gauquelin;

import destiny.astrology.Planet;
import org.junit.Test;

public class GPersonAnglePowerTest
{

  @Test
  public void testSetValue()
  {
    GPersonAnglePower g = new GPersonAnglePower();
    System.out.println(g.getSun() + " : " + g.getSunPower());
    g.setValue(Planet.SUN , "north", 0.99);
    System.out.println(g.getSun() + " : " + g.getSunPower());
  }
}

File: src/test/java/destiny/data/gauquelin/GPersonAnglePowerTest.kt
/**
 * @author smallufo
 * Created on 2009/3/21 at 上午 1:04:35
 */
package destiny.data.gauquelin

import destiny.astrology.Planet
import org.junit.Test

class GPersonAnglePowerTest {

  @Test
  fun testSetValue() {
    val g = GPersonAnglePower()
    println(g.sun + " : " + g.sunPower)
    g.setValue(Planet.SUN, "north", 0.99)
    println(g.sun + " : " + g.sunPower)
  }
}
